static/src/javascripts/test/spec/common/crosswords/helpers.spec.js

```diff
@@ -243,4 +244,253 @@ describe('Helpers', function () {
             expect(helpers.cellsForClue(entriesFixture, clue)).toEqual(expectedCells);
         });
     });
+
+    describe('areCluesInAGroup', function () {
+        var thisEntryFixture = {
```

Comment:
Given that we can run ES6 tests, should we start using `const` and `let` in them wherever possible?